HandEyeCalibration/Transform/Transform.cpp

```diff
@@ -0,0 +1,330 @@
+            auto temp_rollPitchYaw = TransformUtils::rotmat2rpy(rotationMatrix, rotations[i].rot);
+            std::cout << "\nRollPitchYaw" << rotations[i].short_desc << ":\n" << temp_rollPitchYaw << std::endl;
+            fileStorage.write(rotations[i].rpy_name, TransformUtils::eigen2cv(temp_rollPitchYaw));
+            rollPitchYaw[i] = temp_rollPitchYaw;
+        }
+        fileStorage.release();
+
+        /*************************************************************************************************************
+         *      Direction robot to Zivid (any format --> Rotation Matrix)
+         *************************************************************************************************************/
+
+        // Roll, Pitch and Yaw to Rotation Matrix
+        Eigen::Matrix3d rotationMatrixFromRollPitchYaw[4];
+        for(uint32_t i = 0; i < 4; i++)
+        {
+            std::cout << "\nConverting roll-pitch-yaw angles (" << rotations[i].short_desc << ") to rotation matrix"
+                      << std::endl;
+            Eigen::Matrix3d temp_rotationMatrixFromRollPitchYaw =
+                TransformUtils::rpy2rotmat(rollPitchYaw[i], rotations[i].rot);
+            std::cout << "Rotation Matrix:\n" << temp_rotationMatrixFromRollPitchYaw << std::endl;
+
+            // Calculate error in transformation (Rotation Matrix --> Roll-Pitch-Yaw --> Rotation Matrix)
+            std::cout << "\nVerifying Rotation Matrix from Roll-Pitch-Yaw angles (" << rotations[i].short_desc << ")"
+                      << std::endl;
+            TransformUtils::compareRotation(rotationMatrix, temp_rotationMatrixFromRollPitchYaw);
+            TransformUtils::compareRotationExtended(rollPitchYaw[i],
+                                                    temp_rotationMatrixFromRollPitchYaw,
+                                                    rotations[i].rot);
+        }
+
+        // Rotation Vector to Axis-angle
+        std::cout << "\nConverting from rotation vector to axis-angle" << std::endl;
+        axisAngle = Eigen::AngleAxisd(rotationVector.norm(), rotationVector.normalized());
+        axisAngleReordered = TransformUtils::reorderAxisAngle(axisAngle);
+        std::cout << "\nAxis-angle:\n" << axisAngleReordered << std::endl;
+
+        // Axis-Angle to Quaternion
+        std::cout << "\nConverting axis-angle to quaternion" << std::endl;
+        quaternion = Eigen::Quaterniond(axisAngle);
+        quaternionReordered = quaternion.coeffs();
+        std::cout << "\nQuaternion:\n" << quaternionReordered << std::endl;
+
+        // Quaternion to Rotation Matrix
+        std::cout << "\nConverting quaternion to rotation matrix" << std::endl;
+        auto rotationMatrixFromQuaternion = quaternion.toRotationMatrix();
+        std::cout << "\nRotation Matrix:\n" << rotationMatrixFromQuaternion << std::endl;
+
+        // Calculate error in transformation (Rotation Matrix --> Quaternion --> Rotation Matrix)
```

Comment:
Should this be Rotation Matrix --> Quaternion --> Axis-Angle --> Rotation Vector --> Axis-Angle --> Quaternion --> Rotation Matrix?